stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -201,6 +224,30 @@ public boolean onPaymentResult(
         return onPaymentResult(requestCode, resultCode, data, mDefaultPublishableKey, callback);
     }
 
+    /**
+     * Should be called via {@link Activity#onActivityResult(int, int, Intent)}} to handle the
+     * result of a SetupIntent confirmation
+     * (see {@link #confirmSetupIntent(Activity, SetupIntentParams)})
+     */
+    public boolean onSetupResult(int requestCode, int resultCode, @Nullable Intent data,
```

Comment:
should this be `@NonNull Intent data` ?

Reply:
`onActivityResult` from which these would be called has its data params marked as `@Nullable`.  I had removed a `data != null` check because it was technically redundant with the `shouldHandleSetupResult` check, but just noticed it was actually there for the `handleSetupResult` `@NonNull data`.  I re-added the `data != null` check.